frontend/src/query_builder/GuiQueryEditor.jsx

```diff
@@ -399,20 +459,20 @@ export default class GuiQueryEditor extends Component {
             <div className={"GuiBuilder-section GuiBuilder-data flex align-center arrow-right"}>
                 <span className="GuiBuilder-section-label Query-label">Data</span>
                 { this.props.features.data ?
-                    <DataSelector
-                        ref="dataSection"
-                        includeTables={true}
-                        query={this.props.query}
-                        databases={this.props.databases}
-                        tables={this.props.tables}
-                        setDatabaseFn={this.props.setDatabaseFn}
-                        setSourceTableFn={this.props.setSourceTableFn}
-                        isInitiallyOpen={(!this.props.query.database || !this.props.query.query.source_table) && !this.props.isShowingTutorial}
-                    />
-                :
-                    <span className="flex align-center px2 py2 text-bold text-grey">
-                        {this.props.tableMetadata && this.props.tableMetadata.display_name}
-                    </span>
+                  <DataSelector
+                      ref="dataSection"
+                      includeTables={true}
+                      query={this.props.query}
+                      databases={this.props.databases}
+                      tables={this.props.tables}
+                      setDatabaseFn={this.props.setDatabaseFn}
+                      setSourceTableFn={this.props.setSourceTableFn}
+                      isInitiallyOpen={(!this.props.query.database || !this.props.query.query.source_table) && !this.props.isShowingTutorial}
+                  />
+                  :
+                  <span className="flex align-center px2 py2 text-bold text-grey">
+                      {this.props.tableMetadata && this.props.tableMetadata.display_name}
+                  </span>
```

Comment:
naw, the indentation was right before.  it should be 4 spaces from the {} brackets.  your changes here are reducing it to 2 spaces from the bracket to the start of the react element.  you also added 2 spaces in front of the `:` ternary operator, which shouldn't be there either.